sdk/core/azure-core/azure/core/pipeline/transport/__init__.py

```diff
@@ -26,82 +26,131 @@
 
 import sys
 from ._base import HttpTransport, HttpRequest, HttpResponse
-from ._requests_basic import RequestsTransport, RequestsTransportResponse
 
 __all__ = [
     'HttpTransport',
     'HttpRequest',
     'HttpResponse',
-    'RequestsTransport',
-    'RequestsTransportResponse',
 ]
 
 # pylint: disable=unused-import, redefined-outer-name
 try:
-    from ._base_async import AsyncHttpTransport, AsyncHttpResponse
-    from ._requests_asyncio import AsyncioRequestsTransport, AsyncioRequestsTransportResponse
+    from ._requests_basic import RequestsTransport, RequestsTransportResponse
     __all__.extend([
-        'AsyncHttpTransport',
-        'AsyncHttpResponse',
-        'AsyncioRequestsTransport',
-        'AsyncioRequestsTransportResponse'
+        'RequestsTransport',
+        'RequestsTransportResponse',
     ])
+    try:
```

Comment:
Could we add comments to all the the try/except blocks here? The logic is a little hard to follow :)